### PR TITLE
Feature/dom manipulation doc

### LINKED ISSUE #30: Pass all User Stories Functionality
* [x] Import all necessary files
  * [x] Can view list of all Recipes
  * [x] Can click on a recipe and view Directions, ingredients necessary, and total cost
  * [x] Can filter recipes by tags
  * [x] Can search by name + ingredient

### PR DESCRIPTION
**Pull request by Daniel-OC**
## What Features are Being Added?
- Added the innerHTML population of the filter tags
- Integrated the filter by tag functionality into the dom
- The large modal that appears when a recipe card is clicked now populates with the correct information

## What is Being Refactored?
-We removed the innerHTML generation of the modal, opting instead for query selecting the elements we change

## What Do You Need From Your Teammates?
To keep on chugging

## Any Other Notes/Questions/Problems?
Not at this time

### CODE REVIEW

**Review by ectweitmann — APPROVED** [review]
Just need to remove the extra whitespace and add missing semi-colons otherwise it looks good to go.

File: src/index.html [hunk 1]
```diff
@@ -57,35 +57,13 @@ <h1 class="flex">What's Cookin, Good Lookin?</h1>
         </div>
       </section>
       <section class="flex row around main-display lrg-brdr-radius med-marg apricot shadow y-scroll" id="mainDisplay">
-        <article class="flex column sml-brdr-radius shadow">
```
Comment (alexmfritz) [inline src/index.html:60]: The satisfaction of deleting all the HTML after interpolating via a function = chefs kiss!

File: src/scripts.js [hunk 2]
```diff
@@ -43,8 +43,8 @@ const addClass = (elements, rule) => {
     elements.forEach(item => item.classList.add(rule));
 }
 
-const displayCurrentRecipes = (currentRecipes) => {
-  let display = currentRecipes;
+const displayCurrentRecipes = () => {
```
Comment (alexmfritz) [inline src/scripts.js:52]: Great decision not needing to pass anything in here! We can just reassign the global cookbook instance to be whatever the user's favorite recipes are!

File: src/scripts.js [hunk 3]
```diff
@@ -70,17 +70,46 @@ const displayCurrentRecipes = (currentRecipes) => {
...
+    `<section class="flex column align-start eighty-width sml-marg">
+    <section class="flex row">
+      <input class="filter" type="checkbox" name="${tag}" id ="${tag}" />
+      <label for="${tag}">${tag}</label>
+    </section>
+  </section>`
+  })
+}
+
+const getFilterTags = () => {
```
Comment (alexmfritz) [inline src/scripts.js:99]: GREAT work you two!!!

File: src/scripts.js [hunk 4]
```diff
@@ -104,6 +104,26 @@ const getFilterTags = () => {
...
+const createFilterEventListener = () => {
+  const checkBoxes = document.querySelectorAll('.filter');
+  checkBoxes.forEach(checkBox => {
+    checkBox.addEventListener('click', (event) => {
+      displayByTag(event.target.id)
+    })  
+  })
+}
+
+const displayByTag = (tag) => {
```
Comment (alexmfritz) [inline src/scripts.js:119]: Why does ES6 look so dang satisfying?!

**Pull request merged**